aap_compose_dev.yaml

```diff
@@ -124,6 +148,7 @@ services:
     command: |
       bash -c "
         set -e;
+        rm -rf /var/lib/pulp/.migrated;
```

Comment:
restarting containers was not waiting for migrations because file already existed on the volume.